PowerEditor/installer/nativeLang/chinese.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <NotepadPlus>
-    <Native-Langue name="繁體中文/正體中文" filename="chinese.xml" version="7.5.8">
+    <Native-Langue name="繁體中文/正體中文" filename="chinese.xml" version="7.6">
```

Comment:
Now it's 7.6.2, please solve the conflict.

Reply:
Solved. It was 7.6 when this PR opened.